### PR TITLE
style : badge.css

### PR DESCRIPTION
**Pull request by Bibin-Antony**
initial commit

### CODE REVIEW

**Comment by Bibin-Antony** [comment]
image

File: Container/HTML/badge.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Aditi0505) [inline Container/HTML/badge.html:7]: Suggestion: You can add author and description here.

**Review by Aditi0505 — APPROVED** [review]

**Review by maitrakhatri — APPROVED** [review]

File: Container/HTML/badge.html (hunk 1 above)
Comment (Bibin-Antony) [reply Container/HTML/badge.html:7]: done and added to all the other component pages as well.

**Comment by Bibin-Antony** [comment]
Meta tag added and code correction made

**Pull request merged**